### PR TITLE
Understanding Blockchains Interoperability via Cosmos Network

### LINKED ISSUE #6312: [Blockchain] - Understanding blockchains interoperability via cosmos network
#### Proposed title of article
Understanding blockchains interoperability via cosmos network

#### Proposed article introduction
The adoption of crypto is still facing challenges and is still low in numbers compared to the global users of the internet. The main reason for this slow adoption is that blockchains have no interoperability between each other. There have been several approaches devised to ensure the cross-chain interactions between the blockchains, however, most only operate between two blockchains.

The current interoperability techniques seem not to make much impact on network efficiency, upgradability and also sustainability. In an attempt to address these issues, cosmos networks have been developed. In this proposed article we will look at the cosmos network in detail and how it helps to achieve cross-chain communication between blockchains.
 
#### Key takeaways 
At the end of the article the reader will understand:
- what is meant by cosmos network and how it works
- explain the three pillars of cosmos economy
- outline the projects built on cosmos network

#### Article quality
The proposed article is helpful to the readers since it gives insightful ways cosmos networks can ensure interoperability between the blockchains with real-life examples of the projects that are already utilizing it.

#### References
N/A

### PR DESCRIPTION
**Pull request by lewe01**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #6312

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by lalith1403** [comment]
plagiarism check done ✔️

File: content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+#### Band protocol
+The [protocol](https://bandprotocol.com/bandchain) uses web APIs to extract the data. The data is then transferred to other blockchain smart contracts. It has also unlocked a variety of cross-chain functionalities that developers can utilize. The band protocol has partnered with Terra and [Mirror](https://www.mirror.finance/) protocol as their top providers.
+
+#### pSTAKE
+Staked assets are not that liquid in the blockchain. This has not been easy for crypto-investors in years. [pStake](https://pstake.finance/) finance came to change the phenomena. They introduced a liquid staking protocol which is used in their cosmos environment. The protocol uses a [dual token model](https://phemex.com/academy/what-is-the-dual-token-model) whereby stakers can use their assets to issue one to one pegged representatives. It then prospects existing investment opportunities available in the [DeFi](https://www.coinbase.com/learn/crypto-basics/what-is-defi) ecosystem.
```
Comment (wilkistermumbi) [inline content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md:76]:
```suggestion
Staked assets are not that liquid in the blockchain. This has not been easy for crypto-investors in years. [pStake](https://pstake.finance/) finance came to change the phenomena. They introduced a liquid staking protocol which is used in their cosmos environment. The protocol uses a [dual token model](https://phemex.com/academy/what-is-the-dual-token-model) whereby stakers can use their assets to issue one-to-one pegged representatives. It then prospects existing investment opportunities available in the [DeFi](https://www.coinbase.com/learn/crypto-basics/what-is-defi) ecosystem.
```

File: content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+The messages conveyed between the networking layers contain various data ranging from blocks, proposals, and even consensus votes. The [validator nodes](https://learn.radixdlt.com/article/what-is-a-validator-node) work alongside the non-validator nodes. This allows efficient interaction between the nodes, and network peers can quickly access the latest messages and transactions.
+
+#### The application layer
+As discussed earlier, the work of the consensus layer is to perform nodes validation. In contrast, the networking layer transmits messages. The application layer then passes transactions to networking and consensus layers.
+
+In this layer, the transactions happening on the blockchain are determined. They are then forwarded to the consensus layer. The consensus and application layer communication happen through an [Application Blockchain Interface](https://blog.cosmos.network/abci-the-application-blockchain-interface-f1bd8278cdd7) (ABCI). This interface is not limited to a specific programming language. Applications developed to operate on the cosmos network can interact with the base layers of the protocol. This is with disregard of the programming language used.
```
Comment (wilkistermumbi) [inline content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md:46]:
```suggestion
In this layer, the transactions happening on the blockchain are determined. They are then forwarded to the consensus layer. The consensus and application layer communication happens through an [Application Blockchain Interface](https://blog.cosmos.network/abci-the-application-blockchain-interface-f1bd8278cdd7) (ABCI). This interface is not limited to a specific programming language. Applications developed to operate on the cosmos network can interact with the base layers of the protocol. This is with disregard for the programming language used.
```

File: content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+For instance, an ABCI application can assist in deploying the Ethereum codebases on the cosmos network. This will ensure that the challenges of transferring [EVM](https://ethereum.org/en/developers/docs/evm/) to the cosmos are dealt with.
+
+#### IBC protocol
+This [protocol](https://ibcprotocol.org/) handles interoperability between different blockchains in the cosmos network. It also sets up the interaction between blockchains through the [zone and hub model](https://www.preethikasireddy.com/post/how-does-cosmos-work-how-does-it-compare-to-bitcoin-and-ethereum-part-2). Hubs are used to manage different zones. There is indirect interaction between the zones through the use of IBC. The reason for indirect interaction is to ensure that the scalability of the blockchain is not affected.
```
Comment (wilkistermumbi) [inline content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md:57]: The [Inter‑Blockchain Communication Protocol (IBC)](https://ibcprotocol.org/) handles interoperability between different blockchains in the cosmos network. It also sets up the interaction between blockchains through the [zone and hub model](https://www.preethikasireddy.com/post/how-does-cosmos-work-how-does-it-compare-to-bitcoin-and-ethereum-part-2). Hubs are used to manage different zones. There is indirect interaction between the zones through the use of IBC. The reason for indirect interaction is to ensure that the scalability of the blockchain is not affected.

File: content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+#### Cosmos SDK
+[Cosmos SDK](https://v1.cosmos.network/sdk) is another building block of the cosmos network. It significantly improves the innovation around the cosmos network. Developers do not have to develop applications from scratch. They can instead utilize the [cosmos PoS module](https://runtimeverification.com/blog/cosmos-modules-documentation/). Developers can also perform their customizations based on their needs and preferences.
+
+Since Cosmos SDK is modular in design, making blockchains easy to deploy. Developers utilize the cosmos SDK to develop blockchain applications. These applications include [Binance Chain](https://www.binance.com/en), [Terra](https://www.terra.money/), and [Kava](https://www.kava.io/).
```
Comment (wilkistermumbi) [inline content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md:64]:
```suggestion
Cosmos SDK is modular in design, making blockchains easy to deploy. Developers utilize the cosmos SDK to develop blockchain applications. These applications include [Binance Chain](https://www.binance.com/en), [Terra](https://www.terra.money/), and [Kava](https://www.kava.io/).
```

**Comment by wilkistermumbi** [comment]
@lewe01 Some minor edits before we can move on for final review.

**Comment by lewe01** [comment]
Hello @wilkistermumbi. The minor changes requested have been made. You can proceed with the review. 
Thank you

**Comment by wilkistermumbi** [comment]
@lewe01 Great job highlighting the key points about the Cosmos network! This PR will now proceed for final review.

**Comment by LinusMuema** [comment]
Great job on the article @lewe01 

cc: @wilkistermumbi

**Comment by lewe01** [comment]
> Great job on the article @lewe01
> 
> cc: @wilkistermumbi

@LinusMuema Thank you. cc @wilkistermumbi

**Pull request merged**